src/olympia/amo/management/commands/cron.py

```diff
@@ -48,14 +48,17 @@ def handle(self, *args, **options):
 
         module = import_module(path)
 
-        current_millis = datetime.now().timestamp() * 1000
-
+        start_millis = datetime.now().timestamp() * 1000
```

Comment:
Nit: I would have just called it `start_ms` but 🤷

Reply:
I adjusted based on the current naming but I agree